### PR TITLE
B-21054-INT Prime can add second/third addresses to existing shipments

### PR DESCRIPTION
**Pull request by paulstonebraker**
## Summary

Part of the effort to allow prime to view/update/add third addresses (also includes the work to add second addresses). This work allows prime to update existing shipments that don't have second/third addresses. In coming work, we will allow prime to update existing shipments with existing second/third addresses (it uses a different endpoint)

### How to test

1. Create a move with shipments of all types, entering only primary addresses (no second/third addresses)
2. Move the move through the work flow until it gets to prime
3. For all shipment types, click 'Update shipment' and attempt to add second/third addresses where applicable (NTS cannot have second/third addresses added due to a bug which is out of scope for this - an issue has been created)
4. Try and break (but I won't be offended if you don't try too hard in this particular circumstance because prime sim)

NOTE: you can add third addresses before second addresses. Don't. I feel that is out of scope for this work, as its an existing part of the service.

### CODE REVIEW

File: swagger-def/prime_v3.yaml [hunk 1]
```diff
@@ -621,6 +621,15 @@ definitions:
           An optional secondary pickup location near the origin where additional goods exist.
         allOf:
           - $ref: 'definitions/Address.yaml'
+      hasTertiaryPickupAddress:
+        type: boolean
+        x-omitempty: false
+        x-nullable: true
+      tertiaryPickupAddress:
+        description: >
+          An optional second pickup location near the origin where additional goods exist.
```
Comment (pambecker) [inline swagger-def/prime_v3.yaml:630]: update comment to say third

Comment (paulstonebraker) [reply swagger-def/prime_v3.yaml]: Updated!

**Review by pambecker — APPROVED** [review]

File: playwright/tests/office/primesimulator/primeSimulatorFlows.spec.js [hunk 2]
```diff
@@ -148,7 +148,7 @@ test.describe('Prime simulator user', () => {
     await page.locator('input[name="destinationAddress.city"]').fill('Joshua Tree');
     await page.locator('select[name="destinationAddress.state"]').selectOption({ label: 'CA' });
     await page.locator('input[name="destinationAddress.postalCode"]').fill('92252');
-    await page.getByTestId('dropdown').nth(1).selectOption('Home of record (HOR)');
+    await page.getByTestId('dropdown').nth(5).selectOption('Home of record (HOR)');
```
Comment (loganwc) [inline playwright/tests/office/primesimulator/primeSimulatorFlows.spec.js:151]: maybe we rename these data-testid's to something more meaningful?

Comment (paulstonebraker) [reply playwright/tests/office/primesimulator/primeSimulatorFlows.spec.js:151]: but `dropdown` is so semantic

**Pull request merged**